jarbas/frontend/elm/Documents/Inputs/View.elm

```diff
@@ -10,6 +10,42 @@ import Material.Textfield as Textfield
 import Material.Typography as Typography
 import Documents.Inputs.Model exposing (Model, Field, model)
 import Documents.Inputs.Update exposing (Msg(..), update)
+import Internationalization exposing (TranslationId(..), translate)
+import Regex
+
+
+{-| Matches a date un the YYYY-MM-DD format
+
+    >>> matchDate "1970-01-01"
+    True
+
+    >>> matchDate "foo-42-01"
+    False
+
+-}
+matchDate : String -> Bool
+matchDate value =
+    if String.isEmpty value then
+        True
+    else
+        let
+            regex : Regex.Regex
+            regex =
+                Regex.regex
+                    "[\\d]{4}-[\\d]{2}-[\\d]{2}"
```

Comment:
I'm not so worried right now. We have plans to change this UI/UX completely (#61) and probably use a more human-friendly date selector…

Anyway very interesting package this RegexBuilder — totally gonna mess around with it!